apps/wallet/src/state/widget/shared.ts

```diff
@@ -163,7 +165,7 @@ export const mockWidgets: WidgetDocument[] = [
 		networks: [],
 		version: '0.0.1',
 		type: 'Widget',
-		widgetType: WidgetType.GAME,
+		category: WidgetCategory.GAME,
```

Comment:
it is possible to rename this field as it is `category`. But we will try not to rename field because it might cause migration needed.